friend/controller.js

```diff
@@ -0,0 +1,72 @@
+const router = require('express').Router();
+const Friends = require('./friends.js');
+
+router
+    .route('/')
+    .get((req, res) => {
+        Friends.find()
+        .then( friends => {
+            res.status(200).json(friends)
+        
+        })
+        .catch( err => {
+            res.status(500).json({error: "error getting friends"})
+        })
+    })
+    .post((req, res) => {
+        const { body } = req;
+     
+//validates scheme with the New Property
+        const friends = new Friends(req.body);
+        //   
+        friends.save()
+        .then(friends => {
+            res.status(200).json(friends)
+        })
+        .catch(err => {
+            res.status(500).json(err)
```

Comment:
Per the assignment's specs, the error message should be `{ errorMessage: "There was an error while saving the friend to the database." }`